nncf/quantization/algorithms/fast_bias_correction/torch_backend.py

```diff
@@ -0,0 +1,137 @@
+@ALGO_BACKENDS.register(BackendType.TORCH)
+class PTFastBiasCorrectionAlgoBackend(FastBiasCorrectionAlgoBackend):
+
+    TARGET_TYPE_TO_PT_INS_TYPE_MAP = {
+        TargetType.PRE_LAYER_OPERATION: TargetType.OPERATOR_PRE_HOOK,
+        TargetType.POST_LAYER_OPERATION: TargetType.OPERATOR_POST_HOOK,
+    }
+
+    @property
+    def operation_metatypes(self) -> Registry:
+        return PT_OPERATOR_METATYPES
+
+    @property
+    def tensor_processor(self) -> PTNNCFCollectorTensorProcessor:
+        return PTNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> PTTargetPoint:
+        if NNCFGraphNodeType.INPUT_NODE in target_node_name or target_type == TargetType.POST_LAYER_OPERATION:
+            port_id = None
+        if target_type in PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP:
+            target_type = PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP[target_type]
+        return PTTargetPoint(target_type, target_node_name, input_port_id=port_id)
+
+    @staticmethod
+    def create_bias_correction_command(node: NNCFNode,
+                                       bias_value: np.ndarray,
+                                       nncf_graph: NNCFGraph) -> PTBiasCorrectionCommand:
+        return create_bias_correction_command(node, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> PTModelExtractionCommand:
+        return PTModelExtractionCommand(inputs[0])
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape: ReductionShape,
+                                 inplace: bool,
+                                 num_samples: Optional[int] = None,
+                                 window_size: Optional[int] = None) -> PTMeanStatisticCollector:
+        return PTMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_sub_input_output_names(subgraph: NNCFNetwork) -> Tuple[str, str]:
+        # Pytorch does not have name for extracted node
+        return None, None
+
+    @staticmethod
+    def create_input_data(
+        shape: Tuple[int], data: List[torch.Tensor], input_name: str, channel_axis: int
+    ) -> torch.Tensor:
+        blob = torch.zeros(shape, dtype=data[0].dtype)
+        for j, idx in enumerate(np.ndindex(blob.shape[channel_axis])):
+            index = tuple(slice(None) if i != channel_axis else idx for i in range(blob.ndim))
+            blob[index] = data[j]
+        return blob
+
+    @staticmethod
+    def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: NNCFNetwork) -> np.ndarray:
+        return get_bias_value(node, model)
+
+    @staticmethod
+    def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
+        return 0, 0
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: str) -> PTNNCFTensor:
+        return PTNNCFTensor(raw_data)
```

Comment:
Why you don't use `output_name` here? The model can contain multiple outputs and the sub-model too.

Reply:
Because there's no names of output. Sub-model in case of PT is CONV module (weights, bias, FQ for weights). 

Do you know case when `conv` module return multiply or named output?

Reply:
Have you checked the pipeline of the sub-graph creation? Are you sure that sub-graph CAN NOT contain multiple outputs? In the ONNX/OV all outputs are named.

Reply:
> Have you checked the pipeline of the sub-graph creation? Are you sure that sub-graph CAN NOT contain multiple outputs?
Yes, sure that it's not sub-graph. It just one of instance of `Conv` module.


> In the ONNX/OV all outputs are named. 
I know it, but in PT no.

Reply:
No matter. Are you sure that the `Conv` module can contain only one output?

Reply:
>Are you sure that the Conv module can contain only one output?

Ofc. In which backend `conv` layer can return more that one output?
https://pytorch.org/docs/stable/nn.html#convolution-layers